toGeoJSON.js

```diff
@@ -0,0 +1,87 @@
+'use strict'
```

Comment:
shares a lot of code with `readable.js`, but I didn't want to refactor this out (time reason)